.github/workflows/format.yml

```
@@ -0,0 +1,40 @@
+name: Check formatting
+
+on:
+  pull_request:
```

```suggestion
  pull_request:
    branches:
      - master
```

Do you think branch specification is not necessary for examples repo?

---

I would not do branch specification for any repo. It prevents checks when creating a pr that targets another pr

---

I thought about branch spec as about good thing that helps to deal with temporary work merging one pr into another, yes. Now I'm 50/50 about its benefit. It seems that cases of PR to anbother PR are quite rare and only we make them. And I agree that in practice I want to see the results of checks.